tests/aequilibrae/utils/test_create_delaunay_network.py

```diff
@@ -0,0 +1,35 @@
+from unittest import TestCase
+from tempfile import gettempdir
+from os.path import join
+from uuid import uuid4
+from aequilibrae.utils.create_example import create_example
+from aequilibrae.utils.create_delaunay_network import DelaunayAnalysis
+
+
+class TestDelaunayAnalysis(TestCase):
+    def setUp(self) -> None:
+        self.proj = create_example(join(gettempdir(), uuid4().hex))
+
+    def tearDown(self) -> None:
+        self.proj.close()
+
+    def test_create_delaunay_network(self):
+        da = DelaunayAnalysis()
+        with self.assertRaises(ValueError):
+            da.create_network('nodes')
```

Comment:
did you know you can do this?

```suggestion
        self.assertRaises(ValueError, da.create_network, 'nodes'):
```